src/main/java/avd/inf/jdm/rentmycar/controller/dto/CarDTO.java

```diff
@@ -0,0 +1,23 @@
+package avd.inf.jdm.rentmycar.controller.dto;
+
+import avd.inf.jdm.rentmycar.domain.ColorType;
+import avd.inf.jdm.rentmycar.service.CarService;
+import lombok.Getter;
+import lombok.Setter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Getter
+@Setter
+public class CarDTO {
+    private static final Logger log = LoggerFactory.getLogger((CarService.class));
```

Comment:
I'm not familiar with all the logger things but I assume you've tested it and everything works 😅